Cannot build

I'm trying to build a Docker image, but i encounter this problem:
```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Crypto++ (missing: Crypto++_INCLUDE_DIR Crypto++_LIBRARIES)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindCrypto++.cmake:11 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:24 (find_package)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -31,9 +31,10 @@ RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/test
   mariadb-connector-c \
   pugixml
 
+RUN ln -s /usr/lib/libcryptopp.so /usr/lib/libcryptopp.so.5.6
 COPY --from=build /usr/src/forgottenserver/build/tfs /bin/tfs
 COPY data /srv/data/
-COPY LICENSE README.md *.dist *.sql /srv/
+COPY LICENSE README.md *.dist *.sql key.pem /srv/
 
 EXPOSE 7171 7172
 WORKDIR /srv

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,6 +6,7 @@ RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/test
   build-base \
   clang \
   cmake \
+  crypto++-dev \
   gcc \
   gmp-dev \
   luajit-dev \
@@ -24,6 +25,7 @@ FROM alpine:edge
 RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ \
   boost-iostreams \
   boost-system \
+  crypto++ \
   gmp \
   luajit \
   mariadb-connector-c \

